reV/SAM/generation.py

```diff
@@ -1067,18 +1022,15 @@ def cf_mean(self):
 
         return net_power / name_plate
 
-    @property
-    def default(self):
+    @staticmethod
```

Comment:
I still don't get this change. isn't it less efficient to recompute the defaults over and over? Why not add another CLS variable called `DEFAULT_CLS` and then have this:
```
@property
def default(self):
    if self._default is None:
        self._default = self.DEFAULT_CLS.default()
        
        
     return self._default

Reply:
The only place we use the defaults is in the code below, and we retrieve the defaults once and save them, so we dont end up retrieving them again and again anyway (im pretty sure). 

The solution you proposed would still require the class the be instantiated to retrieve the default, which is what i was trying to move away from.

https://github.com/NREL/reV/blob/95082ea79663eff04259cfd9d98b022e60b158d8/reV/generation/base.py#L835-L838

Reply:
Gotcha, this and your earlier comment cleared up my confusion!